config/routes.rb

```diff
@@ -211,6 +211,13 @@
     end
   end
 
+  namespace :referee_interface, path: '/reference' do
+    get '/finish' => 'reference#finish', as: :reference_finish
+    get '/:token/confirmation' => 'reference#confirmation', as: :confirmation
+    get '/:token' => 'reference#feedback', as: :reference_feedback
+    patch '/:token/confirmation' => 'reference#submit_feedback', as: :submit_feedback
+  end
```

Comment:
By default we would log the full URL, including the token. If we'd added the token in the query param it would be automatically filtered out.